ext/date/php_date.stub.php

```diff
@@ -749,3 +746,39 @@ public static function __set_state(array $array): DatePeriod {}
 
     public function getIterator(): Iterator {}
 }
+
+class DateError extends Error
```

Comment:
I'd slap

```
    /**
     * @strict-properties
     */
```

onto each of these. See:

https://github.com/php/php-src/blob/db6840bda42d0fa6e98515eb6f555b0df1e707ee/ext/random/random.stub.php#L171-L173